src/js/firstparties/twitter.js

```diff
@@ -63,13 +76,42 @@ function unwrapTwitterURLsInTimeline() {
   });
 
   // iterate another time to actually replace the link (of Twitter cards, e.g.)
-  findInAllFrames("a[href^='https://t.co/'], a[href^='http://t.co/'").forEach((link) => {
+  findInAllFrames(tcos_all).forEach((link) => {
     if (fixes.hasOwnProperty(link.href)) {
       unwrapTco(link, fixes[link.href]);
     }
   });
 }
 
+function unwrapTwitterPwaURLsInTimeline() {
+  // unwrap profile links
+  document.querySelectorAll(tcos_all).forEach((link) => {
+    // use cached value if possible
+    if (fixes.hasOwnProperty(link.href)) {
+      unwrapTco(link, fixes[link.href]);
+      return;
+    }
+
+    // find span element with link text in it
+    const allSpans = link.getElementsByTagName("span");
+    if (!allSpans) {
+      return;
+    }
+
+    const linkElement = link.getElementsByTagName("span")[0];
```

Comment:
Could we reuse `allSpans` to avoid rerunning the lookup?

Reply:
Oh, of course, missed it.